src/scss/components/email/_index.scss

```diff
@@ -0,0 +1,116 @@
+@import "components/form/config";
+
+hx-email-control {
+  $ctrl-columns: auto 1fr auto;
+  $ctrl-rows: auto auto;
+  $ctrl-row-align: center;
+
+  -ms-grid-columns: $ctrl-columns;
+  -ms-grid-rows: $ctrl-rows;
+  align-items: $ctrl-row-align;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    "label  label   label"
+    "prefix control suffix";
```

Comment:
Update this to a 4-row by 3-column grid:

```
grid-template-areas:
    "label  label   label"
    "prefix control suffix"
    "help   help    help"
    "error  error   error";
```

Reply:
Changes have been done, please review.